readme.md

```diff
@@ -0,0 +1,37 @@
+# SnapShop
+
+---
+
+## Preloaded Data
+### Admin User
+- **Username:** admin  
+- **Password:** admin
+
+### Test User
+- **Username:** test  
+- **Password:** FT5BAk!4SM@vMny
+
+---
```

Comment:
This section is outdated (aea19b9195399f93a8135809599fb2f363813701) and should be removed.